apps/calendar/js/date_l10n.js

```diff
@@ -51,9 +51,13 @@ exports._localizeElement = function(element) {
   var text;
 
   if (format === 'week-hour-format') {
-    text = formatter.format(new Date(date), {
-      dayperiod: '<span class="ampm" aria-hidden="true">$&</span>',
-    });
+    text = formatter.formatToParts(new Date(date)).map(({type, value}) => {
+      switch(type) {
+        case 'dayperiod':
+          return `<span class="ampm" aria-hidden="true">${value}</span>`;
+        default: return value;
+      }
+    }).reduce((string, part) => string + part, '');
```

Comment:
Here and everywhere else where you use `reduce`: you don't have to pass an empty string as the initial value. `reduce` will start with first two elements of the array if you don't, which works fine for string concatenation.